src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/StringItem.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\StringItem as StringItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class StringItem extends StringItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    switch ($field_definition->getName()) {
+      // Random values for orcid need to match the regex ^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$.
+      case 'field_person_orcid':
+        // Generate random string 0000-0000-0000-0000.
+        $orcid = implode('-', array_map(function ($value) {
+          return str_pad(rand(0, pow(10, $value)-1), $value, '0', STR_PAD_LEFT);
+        }, array_fill(0, 4, 4)));
+        // Some orcid's may have a last character of X.
+        $orcid = (rand(0, 10) === 0) ? substr($orcid, 0, -1) . 'X' : $orcid;
```

Comment:
the last symbol would be a checksum apparently, but we don't validate that

Reply:
So, you think I should just leave the last character as an integer?

Reply:
We don't check it anywhere, so probably safe to just generate it.

(Though it's easy to do properly and calculate - https://support.orcid.org/knowledgebase/articles/116780-structure-of-the-orcid-identifier. If we do that, we should also generate in the range 0000-0001-5000-0007 to 0000-0003-5000-0001.)